keyboards/keychron/q1/ansi/keymaps/mkillewald/autocorrect_data.h

```diff
@@ -0,0 +1,176 @@
+// Copyright 2023 QMK
```

Comment:
@drashna I am using two different dictionary files, the 400 entry and the smaller 77 entry file.  I've moved the 400 entry dictionary into my userspace folder because I would like this to be my default for in the future. Unfortunately, at this time, I have only one keymap (the q2) that utilizes the 400 entry dictionaiy.  The Q1 keymaps (using the atmega32u4) are utilizing the smaller 77 entry file because they are near the firmware size limit. So I still have many copies of the same file. Perhaps there is a way to move both dictionary files to the userspace folder and specify which dictionairy file to pull from?

Reply:
you can still use the same file for both.  Eg:
```c
#if defined(__AVR__)
// avr friendly dictionary
#else
// arm dictionary
#endif
```

This would allow you to have one file for both configs.  Either option is fine, though.